charts/mokta/custom_form.haml

```diff
@@ -0,0 +1,155 @@
+%label{ for: "role" } Role
```

Comment:
Not needed for mokta3.  It is now build in.

Reply:
Fantastic! I'll remove it.